README.md

```diff
@@ -220,6 +220,7 @@ Note that:
 | `LOG_LEVEL` | `info` | Level of logging, `debug`, `info`, `warning` or `error` |
 | `LOG_CALLER` | `hidden` | Show caller per log line, `hidden` or `short` |
 | `SHOUTRRR_ADDRESSES` |  | (optional) Comma separated list of [Shoutrrr addresses](https://containrrr.dev/shoutrrr/services/overview/) (notification services) |
+| `SHOUTRRR_PARAMS` |  | (optional) Comma separated list of Shoutrrr parameters (different per service). Set this to `contenttype=text/plain` when using the Generic Shoutrrr service since the default is `application/json`, but ddns-updater sends plain text.  |
```

Comment:
- Added the default `title=DDNS Updater`
- Removed the second part since this is most likely no longer true with the recent update of containerrr/shoutrrr v0.7.0 5fff4f4a564f0b59ba4ab4db16e370fe26f42a39

```suggestion
| `SHOUTRRR_PARAMS` | `title=DDNS Updater` | (optional) Comma separated list of Shoutrrr parameters (different per service). |
```